src/ChangeLog/Changelog.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect, useState} from 'react'
+import ReactMarkdown from 'react-markdown'
+
+function Changelog(){
+    const[posts, setPosts] = useState([])
```

Comment:
This implementation is actually designed to read from multiple md files to truly import "all" posts. Since we will only ever have one md file of significance, we can simplify this whole bit, including the useEffect. This may be a little complicated, so let me know if you have questions.